tests/component/_task_modules/test_triggers.py

```diff
@@ -1,24 +1,38 @@
-from datetime import timedelta
+from datetime import timezone
 
 import pytest
-from hightime import datetime as ht_datetime
+from hightime import datetime as ht_datetime, timedelta as ht_timedelta
 
-from nidaqmx.constants import Timescale
+import nidaqmx
+from nidaqmx.constants import Edge, Timescale, TimestampEvent, TriggerType
+from nidaqmx.error_codes import DAQmxErrors
 from nidaqmx.task import Task
 
 
 @pytest.fixture()
 def ai_voltage_field_daq_task(task, sim_field_daq_device):
-    """Gets AI voltage task."""
+    """Gets AI voltage field daq task."""
     task.ai_channels.add_ai_voltage_chan(sim_field_daq_device.ai_physical_chans[0].name)
     yield task
 
 
+@pytest.fixture()
+def ci_count_edges_task(task) -> nidaqmx.Task:
+    chan = task.ci_channels.add_ci_count_edges_chan(
+        "cdaqChassisTester/_ctr0",
```

Comment:
You should make a `sim_9185_device` fixture to `conftest.py` rather than hardcoding this name

Reply:
Thanks for helping to create a sim_time_aware_9215_device fixture!

Reply:
Thanks for helping to add `sim_9185_device` in your PR!

Reply:
I still need to hardcode this to some degree because 9185's counter channel (_ctr0) cannot be accessed by the typical approach of `sim_9185_device.ci_physical_chans[0].name`. My guess is the _ctr0 is not a physical channel, so `sim_9185_device.ci_physical_chans.channel_names` returns [].

Reply:
Yup, hardcoding the "_ctr0" part is fine. I've done stuff like that in other tests.